docs/src/userguide/numpy_ufuncs.rst

```diff
@@ -1,13 +1,16 @@
-.. highlight:: python
+.. highlight:: cython
```

Comment:
What does this optoin gives us?
Shouldn't we highlight both cython and python?

Reply:
This highlights all examples as cython. If set to python, the cython examples are not highlighted properly. Maybe we can use `:language:` option to set per-example highlight but this works fine too (since cython syntax is kind of subset of python).

Reply:
Yes, the fact that cython is superset if python makes it so some code will be highlighted
But it feels like the thing that should be global for all of the docs and not just here

> Maybe we can use `:language:` option to set per-example highlight

I am not sure but it's reasonable to expect that sphinx-tabs set this filds

The problem is that there are not complete cython highlighter so this is likely the smaller issue